components/src/core/components/Input/Select/select-input.scss

```diff
@@ -196,20 +196,23 @@
 .dropdown-btn {
   display: flex;
   align-items: center;
-  justify-content: space-between;
+  justify-content: space-between !important;
```

Comment:
Remove this generic change and handle only the scenario for this page. Pass it as a style.

Reply:
removed and applied from candidates list configurations as an inline css